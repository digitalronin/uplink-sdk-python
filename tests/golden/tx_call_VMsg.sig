r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 10024262817085143114153833452771322081870872672144051108163381557471621026967
