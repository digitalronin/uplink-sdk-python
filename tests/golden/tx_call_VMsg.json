{
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve", 
    "header": {
        "tag": "TxContract", 
        "contents": {
            "tag": "Call", 
            "contents": {
                "args": [
                    {
                        "tag": "VMsg", 
                        "contents": "Hello world"
                    }
                ], 
                "method": "get", 
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve"
            }
        }
    }, 
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0xMDAyNDI2MjgxNzA4NTE0MzExNDE1MzgzMzQ1Mjc3MTMyMjA4MTg3MDg3MjY3MjE0NDA1MTEwODE2MzM4MTU1NzQ3MTYyMTAyNjk2Nw=="
}